x/ccv/provider/keeper/keeper_test.go

```diff
@@ -396,96 +393,22 @@ func TestVscSendTimestamp(t *testing.T) {
 	require.Empty(t, providerKeeper.GetAllVscSendTimestamps(ctx, chainID))
 }
 
-// TestGetAllConsumerChains tests GetAllConsumerChains behaviour correctness
-func TestGetAllConsumerChains(t *testing.T) {
-	pk, ctx, ctrl, mocks := testkeeper.GetProviderKeeperAndCtx(t, testkeeper.NewInMemKeeperParams(t))
+func TestGetAllRegisteredConsumerChainIDs(t *testing.T) {
+	pk, ctx, ctrl, _ := testkeeper.GetProviderKeeperAndCtx(t, testkeeper.NewInMemKeeperParams(t))
 	defer ctrl.Finish()
 
 	chainIDs := []string{"chain-2", "chain-1", "chain-4", "chain-3"}
```

Comment:
Is there any benefit in having 2 slices? It seems you could only have `expectedChainIDs` and iterate over it to `SetConsumerClientId`s.

Reply:
I want to test that is indeed in the correct order.